pkg/build/controller/strategy/sti.go

```diff
@@ -189,32 +188,29 @@ func (bs *SourceBuildStrategy) CreateBuildPod(build *buildapi.Build) (*v1.Pod, e
 }
 
 func (bs *SourceBuildStrategy) canRunAsRoot(build *buildapi.Build) bool {
```

Comment:
@jim-minter @bparees We're checking whether SA is able to run a pod under root user and trying to create a privileged pod later. Why we don't check whether it able to run a privileged pod? I'm trying to understand a logic here.

Reply:
@php-coder we are checking to see if the SA can run root pods as root because we are going to run a container as root on their behalf later (the assemble container) and we don't want to do that if they aren't supposed to be running pods as root.  privileged doesn't enter into it.